controls/SV-204422.rb

```diff
@@ -39,13 +39,14 @@
       its('lines') { should match_pam_rule("password (required|requisite|sufficient) pam_(unix|pwhistory).so remember=#{reuse_generations}") }
     end
     describe pam('/etc/pam.d/password-auth') do
-      its('lines') {
-        should match_pam_rule("password (required|requisite|sufficient) pam_(unix|pwhistory).soremember=#{reuse_generations}") }
+      its('lines') do
+        should match_pam_rule("password (required|requisite|sufficient) pam_(unix|pwhistory).soremember=#{reuse_generations}")
+      end
```

Comment:
Looks like it changed InSpec syntax here from `{...}` to `do...end`

Reply:
@em-c-rod good catch! This happened because [rubocop](https://docs.rubocop.org/rubocop/cops_style.html#styleblockdelimiters) detected a multi-line block and changed it. Going forward, it'd be helpful to notice any InSpec tests that are on separate lines which could automatically be updated with `do` and `end`. To avoid that, you can disable block delimiters.